tests/e2e/actor_features/actor_features_test.go

```diff
@@ -196,6 +211,108 @@ func TestActorFeatures(t *testing.T) {
+		// Set reminder
+		req := actorReminderOrTimer{
+			Data:    "reminderdata",
+			DueTime: "1s",
+		}
+		reqBody, _ := json.Marshal(req)
+		_, err = utils.HTTPPost(fmt.Sprintf(actorInvokeURLFormat, externalURL, actorID, "reminders", reminderName), reqBody)
+		require.NoError(t, err)
+
+		time.Sleep(secondsToCheckTimerAndReminderResult * time.Second)
+
+		// Reset reminder
+		_, err = utils.HTTPDelete(fmt.Sprintf(actorInvokeURLFormat, externalURL, actorID, "reminders", reminderName))
+		require.NoError(t, err)
+
+		resp, err := utils.HTTPGet(logsURL)
+		require.NoError(t, err)
+		require.True(t, countActorAction(resp, actorID, reminderName) == 1)
+	})
+
+	t.Run("Actor reset reminder.", func(t *testing.T) {
+		// Each test needs to have a different actorID
+		actorID := "1001b"
+
+		// Reset reminder
+		_, err = utils.HTTPDelete(fmt.Sprintf(actorInvokeURLFormat, externalURL, actorID, "reminders", reminderName))
+		require.NoError(t, err)
+		// Set reminder
+		req := actorReminderOrTimer{
+			Data:    "reminderdata",
+			DueTime: "1s",
+			Period:  "5s",
+		}
+		reqBody, _ := json.Marshal(req)
+		_, err = utils.HTTPPost(fmt.Sprintf(actorInvokeURLFormat, externalURL, actorID, "reminders", reminderName), reqBody)
+		require.NoError(t, err)
+
+		time.Sleep(3 * time.Second)
+
+		// Reset reminder (before first period trigger)
+		req.DueTime = "20s"
+		reqBody, _ = json.Marshal(req)
+		_, err = utils.HTTPPost(fmt.Sprintf(actorInvokeURLFormat, externalURL, actorID, "reminders", reminderName), reqBody)
+
+		time.Sleep(10 * time.Second)
+
+		// Delete reminder
+		_, err = utils.HTTPDelete(fmt.Sprintf(actorInvokeURLFormat, externalURL, actorID, "reminders", reminderName))
+		require.NoError(t, err)
+
+		resp, err := utils.HTTPGet(logsURL)
+		require.NoError(t, err)
+		require.True(t, countActorAction(resp, actorID, reminderName) == 1)
+	})
+
+	t.Run("Actor reminder with deactivate.", func(t *testing.T) {
+		// Each test needs to have a different actorID
+		actorID := "1001c"
+
+		// Reset reminder
+		_, err = utils.HTTPDelete(fmt.Sprintf(actorInvokeURLFormat, externalURL, actorID, "reminders", reminderName))
+		require.NoError(t, err)
+		// Set reminder
+		req := actorReminderOrTimer{
+			Data:    "reminderdata",
+			DueTime: "1s",
+			Period:  "1s",
+		}
+		reqBody, _ := json.Marshal(req)
+		_, err = utils.HTTPPost(fmt.Sprintf(actorInvokeURLFormat, externalURL, actorID, "reminders", reminderName), reqBody)
+		require.NoError(t, err)
+
+		sleepTime := secondsToCheckTimerAndReminderResult / 2 * time.Second
+		time.Sleep(sleepTime)
+
+		resp, err := utils.HTTPGet(logsURL)
+		require.NoError(t, err)
+		firstCount := countActorAction(resp, actorID, reminderName)
+		// Min call is based off of having a 1s period/due time, the amount of seconds we've waited, and a bit of room for timing.
+		require.GreaterOrEqual(t, firstCount, 9)
+
+		_, err = utils.HTTPDelete(fmt.Sprintf(actorDeleteURLFormat, externalURL, actorID))
+
+		time.Sleep(sleepTime)
+
```

Comment:
Don't you want to validate that the reminder continues to trigger even after deactivation?

Reply:
It checks that more calls have been made since the deactivation lines 312 and 313.